### PR TITLE
Add shadowsocks-switch@Klavionik

### PR DESCRIPTION
**Pull request by Klavionik**
Hello! 

Long story short: I've found using Shadowsocks under Linux Mint rather hassled and decided to try and implement a simple switch applet. I hope you will find it a useful addition.

### CODE REVIEW

**Review by rcalixte — APPROVED** [review]

File: shadowsocks-switch@Klavionik/files/shadowsocks-switch@Klavionik/icon.png
Comment (rcalixte) [inline shadowsocks-switch@Klavionik/files/shadowsocks-switch@Klavionik/icon.png:1]: @Klavionik The icon needs to be square dimensions and it is currently 62 x 64. If you can fix this, we can merge the new applet. Thank you!

**Review by rcalixte — CHANGES_REQUESTED** [review]

File: shadowsocks-switch@Klavionik/files/shadowsocks-switch@Klavionik/icon.png
Comment (Klavionik) [reply shadowsocks-switch@Klavionik/files/shadowsocks-switch@Klavionik/icon.png:1]: Sorry for the trouble. Fixed the icon and updated the commit.

Comment (Klavionik) [reply shadowsocks-switch@Klavionik/files/shadowsocks-switch@Klavionik/icon.png:1]: I also took a chance to update the screenshot. :)

Comment (rcalixte) [reply shadowsocks-switch@Klavionik/files/shadowsocks-switch@Klavionik/icon.png:1]: No need to apologize! Thanks for your contribution!

**Review by rcalixte — APPROVED** [review]

**Pull request merged**